### PR TITLE
Integrate wandb

### PR DESCRIPTION
**Pull request by nqtabokado**

### CODE REVIEW

File: robo_manip_baselines/policy/act/TrainAct.py [hunk 1]
```diff
@@ -96,12 +144,55 @@ def train_loop(self):
...
+        # Save best model
+        best_ckpt_path = self.save_best_ckpt()
+        if best_ckpt_path:
+            wandb.save(best_ckpt_path)
+
+        wandb.finish()
+
+    # Sweep entrypoint
+    @classmethod
+    def sweep_entrypoint(cls):
```
Comment (mmurooka) [inline robo_manip_baselines/policy/act/TrainAct.py:178]: Is it necessary to define this method in a per-policy class?
If it is sufficient to define it in the `if args.sweep:` block of Train.py, that would be simpler.

**Comment by mmurooka** [comment]
Thanks for the contribution! We're not currently using WandB, but if it becomes truly necessary, we'd be happy to merge it. (Before merging, we'd like to clean it up—for example, by moving as much of the code added to `TrainAct` that is common across all policies into `TrainBase`.)

I'm curious whether sweeps are practically useful. Did using sweeps help you find better hyperparameters?

File: robo_manip_baselines/policy/act/TrainAct.py (hunk 1 above)
Comment (nqtabokado) [reply robo_manip_baselines/policy/act/TrainAct.py]: Thanks! You're right — I’ve moved the sweep logic to Train.py's if args.sweep: block as suggested.

**Comment by nqtabokado** [comment]
> Thanks for the contribution! We're not currently using WandB, but if it becomes truly necessary, we'd be happy to merge it. (Before merging, we'd like to clean it up—for example, by moving as much of the code added to `TrainAct` that is common across all policies into `TrainBase`.)
> 
> I'm curious whether sweeps are practically useful. Did using sweeps help you find better hyperparameters?
Thanks again for the feedback!

I ran a sweep with 10 trials and observed that certain hyperparameter combinations (especially chunk_size, kl_weight, and hidden_dim) made a noticeable difference in validation loss.
image
image
You can see that trial `eager-sweep-1` yielded the best result — achieving the lowest validation loss among all runs.
